packages/avatar/src/avatar.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+
+type AvatarElement = React.ElementRef<'span'>;
+type RootProps = React.ComponentPropsWithoutRef<'span'>;
+
+export interface AvatarProps extends RootProps {
+  src: string;
+  name: string;
+}
+
+export const Avatar = React.forwardRef<AvatarElement, Readonly<AvatarProps>>(
+  ({ src, name, ...props }, forwardedRef) => {
+    avatarImg.backgroundImage = `url(${src})`;
+
+    return (
+      <>
+        <span
+          style={avatarContainer}
+          ref={forwardedRef}
+          {...props}
+        >
+          <label style={avatarLabel}>{abbreviete(name)}</label>
+          <span style={avatarImg}></span>
+        </span>
+      </>
```

Comment:
We can remove the fragment here.
```suggestion
        <span
          style={avatarContainer}
          ref={forwardedRef}
          {...props}
        >
          <label style={avatarLabel}>{abbreviete(name)}</label>
          <span style={avatarImg}></span>
        </span>
```